### PR TITLE
Release notes with Node.js bot 🎉

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: bin/deployment_notification.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+var branch = process.env.TRAVIS_BRANCH;
+var commitAuthor = process.argv[2];
+var commitMessage = process.argv[3];
+var conversationId = '9fe8b359-b9e0-4624-b63c-71747664e4fa';
+var email = process.env.WIRE_WEBAPP_BOT_EMAIL;
+var password = process.env.WIRE_WEBAPP_BOT_PASSWORD;
+var webAppUrl = 'https://app.wire.com/';
+
+switch (branch) {
+  case 'dev':
```
Comment (herrmannplatz) [inline bin/deployment_notification.js:27]: could be an object

File: bin/deployment_notification.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    webAppUrl = 'https://wire-webapp-dev.zinfra.io/auth/?env=prod#login';
+    break;
+  case 'edge':
+    webAppUrl = 'https://wire-webapp-edge.zinfra.io/auth/?env=prod#login';
```
Comment (herrmannplatz) [inline bin/deployment_notification.js]: there is no edge anymore

File: bin/deployment_notification.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+var message =
+  `**New Wire for Web version #$TRAVIS_BUILD_NUMBER online for $TRAVIS_BRANCH.** ᕦ(￣ ³￣)ᕤ`
+  + `\r\n- ${webAppUrl}`
+  + `\r\n- Last commit from: ${commitAuthor}`
+  + `\r\n- Last commit summary: ${commitMessage}`;
+
+var user = new wire.User(email, password).login(false)
+.then(function(service) {
```
Comment (herrmannplatz) [inline bin/deployment_notification.js:45]: we should prefer let over var

File: bin/deployment_notification.sh [hunk 4]
```diff
@@ -4,12 +4,14 @@ AUTHOR_OF_LAST_COMMIT=$(git log | grep Author: | cut -d' ' -f2- | uniq | head -n
 SUMMARY_OF_LAST_COMMIT=$(git log -1 --pretty=%s)
 WEBAPP_URL="https://app.wire.com/"
 
-if [ "$TRAVIS_BRANCH" = "staging" ]; then
-  WEBAPP_URL="https://wire-webapp-staging.zinfra.io/auth/?env=prod#login"
+if [ "$TRAVIS_BRANCH" = "dev" ]; then
+  WEBAPP_URL="https://wire-webapp-dev.zinfra.io/auth/?env=prod#login"
 elif [ "$TRAVIS_BRANCH" = "edge" ]; then
   WEBAPP_URL="https://wire-webapp-edge.zinfra.io/auth/?env=prod#login"
```
Comment (herrmannplatz) [inline bin/deployment_notification.sh:10]: we can remove edge from here as well

File: bin/deployment_notification.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+const wire = require('wire-webapp-core');
```
Comment (herrmannplatz) [inline bin/deployment_notification.js:3]: 'use strict'

File: bin/deployment_notification.js (hunk 3 above)
Comment (lipis) [reply bin/deployment_notification.js:45]: leeeeeeet

File: bin/deployment_notification.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  return service.conversation.sendTextMessage(conversationId, message);
+})
+.then(function(service) {
+  return service.user.logout();
+})
+.then(function() {
+  return process.exit(0);
+})
+.catch(function(error) {
+  console.log('Error: ' + error.message);
```
Comment (lipis) [inline bin/deployment_notification.js]:
```js
console.log('Error:', error.message)
```

**Pull request merged**